libs/components/modals/src/lib/modules/modal/modal-header.component.html

```diff
@@ -3,6 +3,8 @@
   [skyThemeClass]="{
     'sky-font-display-3': 'modern'
   }"
+  skyTrim
 >
   <ng-content></ng-content>
 </h1>
+<ng-content select=".sky-control-help"></ng-content>
```

Comment:
You need to collapse the whitespace around the `ng-content` as well as wrap it in a `span` with a `sky-control-help-container` class. See [description list](https://github.com/blackbaud/skyux/blob/main/libs/components/layout/src/lib/modules/description-list/description-list-term.component.html) for an example.